opengrok-indexer/src/main/java/org/opengrok/indexer/index/Indexer.java

```diff
@@ -161,9 +163,36 @@ public static void main(String[] argv) {
 
         boolean createDict = false;
 
+        int CONNECT_TIMEOUT = 1000;  // in milliseconds
+
         try {
             argv = parseOptions(argv);
 
+            if (webappURI != null) {
```

Comment:
I'd move this check inside a separate method inside the `HostUtil` class. I think the `main` method is already too long.